docs/grammar.md

```diff
@@ -262,8 +262,8 @@ Now, an ordered choice in the parentheses is optional.
 
 * **Unordered group** is a special kind of a sequence. Syntactically it is
   similar to a repetition. It is specified by the `#` operator and must be
-  applied to sequences. This operator will match each element of the sequence in
-  an arbitrary order:
+  applied to either sequences or ordered choices. This operator will match each
+  element of the sequence in an arbitrary order:
```

Comment:
...will match each element of the sequence +"or the ordered choice"  in an arbitrary order: